lib/ilp.js

```diff
@@ -96,10 +96,15 @@ class Ilp {
 
     this.log.info('Connection created successfully')
 
-    await this.sendMoney({ ilpAmount: ilpScaledAmount, connection })
+    let { success, remainingAmount } = await this.sendMoney({ ilpAmount: ilpScaledAmount, connection })
+    // Reset remaining amount to millicents, where as ILP operates in nanodollars
+    if (remainingAmount > 0) remainingAmount = remainingAmount / 10000
 
     this.log.info('Closing connection to ILP connector...')
     await connection.end()
+
+    // Pass along success and remaining amount to caller so they can handle it
+    return { success: success || true, remainingAmount: remainingAmount || 0 }
```

Comment:
this means it's always success:true, which is not what we want

Reply:
good catch, done and added test case